Packaging of Syncplay in a single executable .app file for macOS

alby128 opened the pull request:
Gives a way for users on macOS to run Syncplay without evening opening their terminal once. Creation of the .app and packaging of it in a distributable .dmg are handled by Travis CI. Appveyor support is also experimentally added for Windows. Binaries deployment is done through Bintray.

Also, solves issues #87 and #89. In addition, it adds an About dialog on all operative systems that include information about Syncplay, its license and the license of its dependencies.

Et0h commented:
Does  "git clone -b macOS_app https://github.com/alby128/syncplay.git syncplay_app" needed to be amended to use official Syncplay repo?

Et0h commented:
And does the bintray name still want to be "1.5.0_test"?

alby128 commented:
Fixed and fixed. Changed bintray version to 1.5.0, even though bintray will not be used (probably) for officially distributing the .dmg file. If we decide to do so, we should probably create a bintray account for Syncplay.
